stock_location_restrict_group/models/stock_location.py

```diff
@@ -26,7 +26,6 @@ def _compute_restricted_group(self):
         for location in self.filtered(lambda l: l.group_restricted):
             quants = self.env['stock.quant'].search(
                 [('location_id', '=', location.id)])
-            location.restricted_group = False
             rest_group = quants.mapped('history_ids').filtered(
```

Comment:
I think leaving the field as not store is sufficent for the purpose.
Maybe if we want to add a search field on location tree if we want to find the location for a particular p. group. What do you think about?